harvester/harvester-installer
# Enhancement: Add IP mask field to Network Panel

Add IP mask field to provide better network configuration options:
- Display calculated netmask when CIDR notation is used
- Allow manual netmask input for non-CIDR IP addresses

**Problem:**
Allow users to specify the address as either `<address>/<netmask>` or CIDR. 

**Solution:**
Display calculated netmask when CIDR notation is used. And validate the netmask input for non-CIDR IP addresses

**Related Issue:**
https://github.com/harvester/harvester/issues/6295

**Test plan:**
1. Install Harvester
2. Proceed to Network Configuration Panel
3. Check the existence of `IPv4 Mask` field
4. Input a CIDR format in `IPv4 Address` field and check if the netmask generated correctly in the `IPv4 Mask` field
5. Input a non-CIDR format IP and manually input the netmask with values to validate if the netmask applies or not.
